packages/blaze/builtins.js

```diff
@@ -226,6 +226,11 @@ Blaze.Each = function (argFunc, contentFunc, elseFunc) {
         });
       },
       changedAt: function (id, newItem, oldItem, index) {
```

Comment:
In [my example](https://github.com/mitar/test-blaze-array) I could not get to this issue. If I instrument `changedAt` method to print out every time is called, and then I click "add 5" button, this is not called at all, not for 5 or any other value in the Minimongo database.